reference/events.rst

```diff
@@ -0,0 +1,244 @@
+Symfony Framework Events
+========================
+
+Kernel Events
+-------------
+
+Each event dispatched by the kernel is a subclass of
+:class:`Symfony\\Component\\HttpKernel\\Event\\KernelEvent`. This means that
+each event has access to the following information:
+
+:method:`Symfony\\Component\\HttpKernel\\Event\\KernelEvent::getRequestType`
+    Returns the *type* of the request (``HttpKernelInterface::MASTER_REQUEST`` or
+    ``HttpKernelInterface::SUB_REQUEST``).
+
+:method:`Symfony\\Component\\HttpKernel\\Event\\KernelEvent::getKernel`
+    Returns the Kernel handling the request.
+
+:method:`Symfony\\Component\\HttpKernel\\Event\\KernelEvent::getRequest`
+    Returns the current ``Request`` being handled.
+
+.. _kernel-core-request:
+
+``kernel.request``
+~~~~~~~~~~~~~~~~~~
+
+**Event Class**: :class:`Symfony\\Component\\HttpKernel\\Event\\GetResponseEvent`
+
+The goal of this event is to either return a ``Response`` object immediately
```

Comment:
I also saw an instance of just `response object` above, either set this as the one above or even those in a way